R/plot-histogram.R

```diff
@@ -109,21 +109,33 @@ plotHistogram <- function(data = NULL,
   if (length(dataMapping$bins) > 1) {
     edges <- dataMapping$bins
   }
-  # Manage ggplot aes_string property depending on stack and frequency options
+  # Manage ggplot aes property depending on stack and frequency options
   # geom_histogram can use computed variables defined between two dots
   # see https://ggplot2.tidyverse.org/reference/geom_histogram.html for more info
-  yAes <- "..count.."
+  mapping <- ggplot2::aes(
+    x = .data[[mapLabels$x]],
+    y = after_stat(count),
+    fill = .data[[mapLabels$fill]]
+  )
 
   if (dataMapping$frequency) {
     # If histogram bars are not stacked, calculate frequency within each data groups
     # Since there is no direct computed variable
     # ncount variable is scaled by binwidth*dnorm(0) to get an area of ~1
-    yAes <- paste0("..ncount..*max(..width..)*", stats::dnorm(0))
+    mapping <- ggplot2::aes(
+      x = .data[[mapLabels$x]],
+      y = after_stat(ncount * max(width) * dnorm(0)),
+      fill = .data[[mapLabels$fill]]
+    )
     if (dataMapping$stack) {
       # If histogram bars are stacked,
       # Calculate overall frequency as count per bin / total
       # This results in same histogram shapes no matter the data groups
-      yAes <- "..count../sum(..count..)"
+      mapping <- ggplot2::aes(
+        x = .data[[mapLabels$x]],
+        y = after_stat(coun / sum(count)),
```

Comment:
Is `coun` a mistype here?

Reply:
I think using {lintr} would be great. There's already a Github action ready to go [here](https://github.com/r-lib/actions/tree/old/examples#lint-workflow).

We would need to customize it a little and define which [linter](https://lintr.r-lib.org/reference/linters.html) we want to apply. But that depends on the scope of the verification we want to implement.

Maybe we can start with the most basic level: make sure that the code can be evaluated. I can make some test if required.